### PR TITLE
Test/backup all ok.rec

### PR DESCRIPTION
**Pull request by PavelShilin89**

### CODE REVIEW

**Review by donhardman — CHANGES_REQUESTED** [review]
It looks legit, but please can you rebase from the latest master branch and squash all commits before we can merge it.

File: tests/backup-all-ok.rec [hunk 1]
```diff
@@ -72,7 +72,7 @@ mysql -h0 -P9306 -e "flush ramchunk test; select * from test.@files;"
 |   10 | /var/lib/manticore/test/test.0.spds   | /var/lib/manticore/test/test.0.spds   |     64 |
 |   11 | /var/lib/manticore/test/test.0.spknn  | /var/lib/manticore/test/test.0.spknn  |    456 |
 |   12 | /var/lib/manticore/test/test.0.spt    | /var/lib/manticore/test/test.0.spt    |     41 |
-|   13 | /var/lib/manticore/test/test.0.spidx  | /var/lib/manticore/test/test.0.spidx  |    309 |
+|   13 | /var/lib/manticore/test/test.0.spidx  | /var/lib/manticore/test/test.0.spidx  |    149 |
```
Comment (djklim87) [inline tests/backup-all-ok.rec:75]: Should we store digits here? Maybe just use regex?

File: tests/backup-all-ok.rec [hunk 2]
```diff
@@ -104,7 +104,7 @@ buddy: %{SEMVER}
 [93m%{DATETIME}[0m [Info]    [92mOK[0m
 [93m%{DATETIME}[0m [Info]   rt_without_columnar (rt) [2.789K]...
 [93m%{DATETIME}[0m [Info]    [92mOK[0m
-[93m%{DATETIME}[0m [Info]   test (rt) [130.574K]...
+[93m%{DATETIME}[0m [Info]   test (rt) [130.418K]...
```
Comment (djklim87) [inline tests/backup-all-ok.rec:107]: Regex?

**Review by djklim87 — CHANGES_REQUESTED** [review]

File: tests/backup-all-ok.rec (hunk 1 above)
Comment (PavelShilin89) [reply tests/backup-all-ok.rec]: done

**Pull request merged**